[The test failure report](https://github.com/mlflow/mlflow/runs/6366020248?check_suite_focus=true) indicates `Path("foo/bar").resolve().as_uri()` failed:

```
__________________ test_set_tracking_uri_with_path[relative] __________________

tmp_path = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-0/test_set_tracking_uri_with_pat1')
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x000001874D42C6C8>
absolute = False

    @pytest.mark.parametrize("absolute", [True, False], ids=["absolute", "relative"])
    def test_set_tracking_uri_with_path(tmp_path, monkeypatch, absolute):
        monkeypatch.chdir(tmp_path)
        path = Path("foo/bar")
        if absolute:
            path = tmp_path / path
        with mock.patch("mlflow.tracking._tracking_service.utils._tracking_uri", None):
            set_tracking_uri(path)
>           assert get_tracking_uri() == path.resolve().as_uri()

absolute   = False
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x000001874D42C6C8>
path       = WindowsPath('foo/bar')
tmp_path   = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-0/test_set_tracking_uri_with_pat1')

D:\a\mlflow\mlflow\tests\tracking\_tracking_service\test_utils.py:374: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = WindowsPath('foo/bar')

    def as_uri(self):
        """Return the path as a 'file' URI."""
        if not self.is_absolute():
>           raise ValueError("relative path can't be expressed as a file URI")
E           ValueError: relative path can't be expressed as a file URI

self       = WindowsPath('foo/bar')

c:\hostedtoolcache\windows\python\3.7.9\x64\lib\pathlib.py:739: ValueError
```

--- a/tests/tracking/_tracking_service/test_utils.py
+++ b/tests/tracking/_tracking_service/test_utils.py
@@ -371,4 +371,4 @@ def test_set_tracking_uri_with_path(tmp_path, monkeypatch, absolute):
         path = tmp_path / path
     with mock.patch("mlflow.tracking._tracking_service.utils._tracking_uri", None):
         set_tracking_uri(path)
-        assert get_tracking_uri() == path.resolve().as_uri()
+        assert get_tracking_uri() == path.absolute().resolve().as_uri()

